migrations/2018-01-08-072612_status_uri/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE statuses ADD COLUMN uri TEXT;
```

Comment:
random thought semi-unrelated to review: we should probably pick one of {`TEXT`, `VARCHAR`} for uris and be done with it; I know I made uri ` VARCHAR` on the `accounts` table.